res/css/views/rooms/_E2EIcon.scss

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2019 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_E2EIcon {
+    width: 25px;
+    height: 25px;
+    mask-repeat: no-repeat;
+    mask-position: center 0;
+    margin: 0 9px;
+}
+
+.mx_E2EIcon_verified {
+    mask-image: url('$(res)/img/feather-icons/e2e/lock-verified.svg');
```

Comment:
These are all icons Nad made, right? Is `feather-icons` the right place?

@ara4n recently [told me](https://github.com/matrix-org/matrix-react-sdk/pull/2540#discussion_r252491900) that `feather-icons` is meant to be only icons from https://feathericons.com/, and I filed https://github.com/vector-im/riot-web/issues/8348 to clean up existing ones.

If you want to leave them there for now though and let https://github.com/vector-im/riot-web/issues/8348 sort it out, that's okay with me.

Reply:
ah ok, sounds good. thanks